### PR TITLE
1078 create unit tests for has attr

### PR DESCRIPTION
**Pull request by VeronikaRevjakina**
Temporary
Do not merge :)

### CODE REVIEW

**Comment by VeronikaRevjakina** [comment]
Need review

File: jdi-light/src/test/java/mockitotests/HasAttrMockTests.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        Mockito.doReturn(actualMapAttr).when(uiElement).attrs();
+
+        try {
+            uiElement.is().hasAttr(wrongAttr);
+        } catch (RuntimeException exp) {
+            String expMessage = exp.getMessage();
+            assertTrue(expMessage.contains(exceptionMessageFirstPart + " \"" + wrongAttr + "\""));
+            assertTrue(expMessage.contains(exceptionMessageSecondPart));
+            for (String key : actualMapAttr.keys()) {
+                System.out.println("was  \"" + key + "\"");
```
Comment (mf-code) [inline jdi-light/src/test/java/mockitotests/HasAttrMockTests.java]: please remove this log

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light/src/test/java/mockitotests/HasAttrMockTests.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4451777&bid=15047462)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light/src/test/java/mockitotests/HasAttrMockTests.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4451777&bid=15047462)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- jdi-light/src/test/java/mockitotests/HasAttrMockTests.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4451777&bid=15047462)

File: jdi-light/src/test/java/mockitotests/HasAttrMockTests.java (hunk 1 above)
Comment (VeronikaRevjakina) [reply jdi-light/src/test/java/mockitotests/HasAttrMockTests.java]: @mf-code Fixed , sorry forgot about that :)
I checked all requirements in PR review file, it fits, but assertContains needed to be used (not JDI assert)
And Travis is still falling

**Pull request merged**